### PR TITLE
bpo-38218: Corrected syntax for return annotation

### PR DESCRIPTION
**Pull request by pluradj**
Signed-off-by: Jason Plurad <pluradj@us.ibm.com>

https://bugs.python.org/issue38218

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@pluradj

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

**Review by tirkarthi — APPROVED** [review]
LGTM. Thanks for the catch.

File: Doc/tutorial/errors.rst [hunk 1]
```diff
@@ -400,7 +400,7 @@ If a :keyword:`finally` clause is present, the :keyword:`finally` clause will ex
 
 For example::
 
-   >>> def bool_return(): -> bool:
+   >>> def bool_return() -> bool:
```
Comment (serhiy-storchaka) [inline Doc/tutorial/errors.rst]: I do not think this function should use a return annotation. No other code or examples (except related to the typing module) use annotations. I think it was intended as
```
def bool_return():  #  -> bool
```
Or maybe ` -> bool:` should be removed at all.

Comment (pluradj) [reply Doc/tutorial/errors.rst]: agreed, happy to make that adjustment.

Comment (ammaraskar) [reply Doc/tutorial/errors.rst]: Seconding the

> Or maybe -> bool: should be removed at all.

suggestion.

**Comment by pluradj** [comment]
I've submitted the CLA just now.

**Review by serhiy-storchaka — APPROVED** [review]
Although this is an example of non-recommended code.

**Comment by matrixise** [comment]
Thank you for your PR, because it's a type-documentation PR, I merge it into master, 3.8 and 3.7.
Have a nice day and thanks again. Thank you to @serhiy-storchaka and @tirkarthi for the reviews and approvals.

**Pull request merged**